app/assets/stylesheets/application.scss

```diff
@@ -538,11 +513,43 @@ span.glyphicon.host-status {
   word-break: break-all;
 }
 
-.ms-container {
-  background: transparent image-url('switch.png') no-repeat 270px 80px;
-}
-
 .big {
   font-size: 3rem;
   color: #797979;
 }
+
+html.turbolinks-progress-bar::before {
+  height: 2px !important;
+}
+
+.white-header {
+  background-color: white !important;
+  th{
+    background-color: white !important;
+  }
+}
+
+#proxy-tab {
+  margin-bottom: 0px;
+}
+
+.proxy-content.tab-content {
+  padding: 19px;
+  min-height: 0px;
+  border: 1px solid #ddd;
+  border-radius: 2px;
+  border-top: 0px;
+  float: left;
+  width: 100%;
+}
+
+.spinner-placeholder{
+  position: absolute;
+}
+
+.spinner-label{
+  display: block;
+  float: left;
+}
+
+@import "/**/*";
```

Comment:
should this be last? if it serve a specific purpose maybe comment on it so we keep it last?